.travis.yml

```diff
@@ -18,6 +18,10 @@ matrix:
     # language: java causes Travis to automatically run Gradle tests based on the
     # presence of the appropriate build files.
     - language: java # Gradle (used automatically)
+      install:
```

Comment:
can you rebase against master? i think this is from an earlier commit

Reply:
Done.